enterprise/ha/src/main/java/org/neo4j/kernel/ha/id/HaIdGeneratorFactory.java

```diff
@@ -73,6 +73,7 @@ public IdGenerator open( FileSystemAbstraction fs, File fileName, int grabSize,
             initialIdGenerator = localFactory.open( fs, fileName, grabSize, idType, highId );
             break;
         case SLAVE:
+            fs.deleteFile( fileName );
```

Comment:
I don't see why we need this, shouldn't we always go through `switchToSlave` when we join the cluster as slaves?